db.js

```diff
@@ -0,0 +1,118 @@
+require("dotenv").config()
+const { Client } = require("pg");
+
+const client = new Client({
+  user: process.env.DB_USER,
+  host: process.env.DB_HOST,
+  database: process.env.DB_NAME,
+  password: process.env.DB_PASSWORD,
+  port: Number(process.env.DB_USER),
+});
+
+client
+  .connect()
+  .then(() => console.log("Connected to database"))
+  .catch((err) => console.error("Error connecting to database: ", err.stack));
+
+async function closeConnection() {
+  client
+    .end()
+    .then(() => console.log("Connection ended"))
+    .catch((err) => console.error("Error closing connection: ", err.stack));
+}
+
+async function fetchTodos() {
+  try {
+    res = await client.query("SELECT * FROM todo");
+    if (res.rowCount === 0) {
+      console.log("No tasks found");
+      return;
+    } else {
+      res.rows.forEach((row) =>
+        console.log(`${row.id}. ${row.task} [${row.done ? "x" : ""}]`)
+      );
+    }
```

Comment:
if you're using a return, there is no need for you to add an else block

Reply:
yessir